lib/pages/chat_list/chat_list_item.dart

```diff
@@ -201,140 +205,189 @@ class ChatListItem extends StatelessWidget {
+                  }
+
+                  final isMentionned = room.lastEvent!.room.getMention(snapshot.data!) == Matrix.of(context).client.userID;
+                  return AnimatedContainer(
                     duration: FluffyThemes.animationDuration,
                     curve: FluffyThemes.animationCurve,
                     padding: const EdgeInsets.symmetric(horizontal: 7),
-                    height: unreadBubbleSize,
-                    width: room.notificationCount == 0 &&
-                            !unread &&
-                            !room.hasNewMessages
-                        ? 0
-                        : (unreadBubbleSize - 9) *
-                                room.notificationCount.toString().length +
-                            9,
+                    height: ChatListItemStyle.mentionIconWidth,
+                    width: isMentionned ? ChatListItemStyle.mentionIconWidth : 0,
                     decoration: BoxDecoration(
-                      color: room.highlightCount > 0 ||
-                              room.membership == Membership.invite
-                          ? Colors.red
-                          : room.notificationCount > 0 || room.markedUnread
-                              ? Theme.of(context).colorScheme.primary
-                              : Theme.of(context).colorScheme.primaryContainer,
-                      borderRadius:
-                          BorderRadius.circular(AppConfig.borderRadius),
+                      color: Theme.of(context).colorScheme.primary,
+                      borderRadius: BorderRadius.circular(AppConfig.borderRadius),
                     ),
                     child: Center(
-                      child: room.notificationCount > 0
+                      child: isMentionned
                           ? Text(
-                              room.notificationCount.toString(),
-                              style: TextStyle(
-                                color: room.highlightCount > 0
-                                    ? Colors.white
-                                    : room.notificationCount > 0
-                                        ? Theme.of(context)
-                                            .colorScheme
-                                            .onPrimary
-                                        : Theme.of(context)
-                                            .colorScheme
-                                            .onPrimaryContainer,
-                                fontSize: 13,
-                              ),
-                            )
+                                  '@',
+                                  style: TextStyle(
+                                    color: isMentionned
+                                        ? Theme.of(context).colorScheme.onPrimary
+                                        : Theme.of(context).colorScheme.onPrimaryContainer,
+                                    fontSize: 12,
```

Comment:
<img width="228" alt="Screenshot 2023-05-26 at 14 10 34" src="https://github.com/linagora/twake-on-matrix/assets/43041967/eca6454b-ed3d-48dd-8db1-46d5ce1d5397">

`Theme.of(context).textTheme.labelMedium`